docs/dev/setup/kubernetes.rst

```diff
@@ -0,0 +1,353 @@
+---------------------------
+The Artemis image will be stored and managed in DockerHub. Kubernetes will pull it from there and deploy it afterwards.
+After you log in to your `DockerHub <https://hub.docker.com/>`__ account you can create as many public repositories as you want.
+In order to create a repository you need to select the ``Create repository`` button.
+
+
+**DockerHub:**
+
+.. figure:: kubernetes/dockerhub.png
+   :align: center
+
+Then fill in the repository name with ``artemis``. The use the ``Create`` button and you repository is created.
+
+.. figure:: kubernetes/dockerhub_create_repository.png
+   :align: center
+
+Configure Docker ID (username)
+------------------------------
+The username in DockerHub is called Docker ID. You need to set your Docker ID in the ``artemis-deployment.yml`` resource so that Kubernetes knows where to pull the image from.
+Open the ``src/main/kubernetes/artemis/deployment/artemis-deployment.yml`` file and edit
+
+   ::
+
+      template:
+         spec:
+         containers:
+            image: <DockerId>/artemis
+
+and replace <DockerId> with your docker ID in DockerHub
+
+i.e. it will look like this:
+
+   ::
+
+      template:
+         spec:
+         containers:
+            image: mmehmed/artemis
+
+
+
+Configure Artemis resources
+---------------------------
+
+In order to run Artemis, you need to configure the Artemis' User Management, Version Control and Continuous Integration. You can either run it with Jira, Bitbucket, Bamboo or Jenkins, Gitlab.
+Make sure to configure the the ``src/main/resources/config/application-artemis.yml`` file with the proper configuration for User Management, Version Control and Continuous Integration.
+If you want to configure Artemis with ``Bitbucket, Jira, Bamboo`` continue with ``Configure Bitbucket, Jira, Bamboo`` or if you want to configure Artemis with local user management and no programming exercise continue with ``Configure Local User Mangement``.
+
+Configure Bitbucket, Jira, Bamboo
+#################################
+
+If you run Artemis with Bitbucket, Jira, Bamboo you have to deploy them on Kubernetes. Since Artemis will be deployed on a cluster, local connections to Jira, Bitbucket, Bamboo will not work. 
+Therefore you have to deploy them on Kubernetes or to set connection to existing staging or production deployments, if any. The latter can be done only if you have admin access to them.
+In order to deploy Bitbucket, Jira, Bamboo on Kubernetes use the following documentation: `Bitbucket, Jira, Bamboo <https://docs.artemis.ase.in.tum.de/dev/setup/bamboo-bitbucket-jira/>`__.
+Once you are done continue with the next step ``Build Artemis``
+
+Configure Local User Mangement
+##############################
+
+If you want to run with local user management and no programming exercises setup follow the steps: 
+
+1. Go to the ``src/main/resources/config/application-artemis.yml`` file, and set use-external in the user-management section to false.
```

Comment:
you could add a hint, that if there is an additional `application-local.yml` as described in the [setup doc](https://artemis-platform.readthedocs.io/en/latest/dev/setup/#server-setup) 
<img width="745" alt="Bildschirmfoto 2021-08-25 um 10 51 10" src="https://user-images.githubusercontent.com/41366522/130759876-c950c71a-d388-4954-be3c-7dc4f3b0ceeb.png">
it has to be changed there as well (acutually, this did not work for me too) -> what did the job was to add the environment variable, as you suggested, in rancher:
<img width="1911" alt="Bildschirmfoto 2021-08-25 um 10 52 29" src="https://user-images.githubusercontent.com/41366522/130760035-d44a1975-db0e-4fcb-b36b-5aa7c32b8619.png">